### PR TITLE
add support for laravel 7

### PR DESCRIPTION
**Pull request by norbybaru**

### CODE REVIEW

File: src/StateWorkflowServiceProvider.php [hunk 1]
```diff
@@ -49,22 +49,40 @@ protected function configPath()
...
     }
 
+    /**
+     * Return migrations path.
+     *
+     * @return string
+     */
+    private function migrationPath()
+    {
+        return __DIR__.'/../database/migrations';
```
Comment (datashaman) [inline src/StateWorkflowServiceProvider.php:59]:
```
database_path('migrations');
```

File: src/StateWorkflowServiceProvider.php [hunk 2]
```diff
@@ -49,22 +49,40 @@ protected function configPath()
...
+
     /**
      * Publish config file.
      */
     protected function publishConfig()
     {
-        $this->publishes([
-            $this->configPath() => config_path('workflow.php'),
-        ], 'config');
+        if ($this->app->runningInConsole()) {
```
Comment (datashaman) [inline src/StateWorkflowServiceProvider.php:67]: Why do this?

Comment (norbybaru) [reply src/StateWorkflowServiceProvider.php:67]: publishing of assets should only happen via artisan command. its newer approach Laravel is using

Comment (datashaman) [reply src/StateWorkflowServiceProvider.php:67]: The service provider should only use this if you ask for the files to be published with `vendor:publish`. Wondering why this is being done.

File: src/StateWorkflowServiceProvider.php (hunk 1 above)
Comment (norbybaru) [reply src/StateWorkflowServiceProvider.php:59]: hmm this will not work for package. only works on the framework

Comment (atimbang-ROAM) [reply src/StateWorkflowServiceProvider.php:59]: @norbybaru @datashaman the multi tenant package i'm using moves the migration files into a folder
`database_path('migrations/tenant')` any idea how we can support these two folder structure?

Comment (atimbang-ROAM) [reply src/StateWorkflowServiceProvider.php:59]: BTW I'll send in a PR for laravel 8 support, working on upgrading the Laravel version in property project

**Pull request merged**